### PR TITLE
misc: drop redundant dependencies to speed-up build times

### PR DESCRIPTION
**Pull request by isidentical**
### Integration tests

Adapter to test:
- postgres

Python version to test:

- 3.8

### CODE REVIEW

**Review by mederka — APPROVED** [review]
ATL: how big is the speed-up?

**Comment by isidentical** [comment]
> ATL: how big is the speed-up?

It is nothing major, just shaving a couple of seconds from the installation time. Still an easy/simple win.

File: pyproject.toml [hunk 1]
```diff
@@ -21,17 +21,20 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.7.1, <3.11"
 pandas = "^1.3.4"
-arrow = "^1.2.0"
```
Comment (chamini2) [inline pyproject.toml:24]: are we sure about this one?

**Comment by chamini2** [comment]
Started this to make sure: https://github.com/fal-ai/fal/actions/runs/3113115123

File: pyproject.toml (hunk 1 above)
Comment (isidentical) [reply pyproject.toml:24]: `arrow` on PyPI seems to be an unrelated library (to actual Apache Arrow) that does date processing, https://pypi.org/project/arrow/. I grepped the codebase, and seems like we don't use it at all. The PyArrow dependency is still here by the way (just right under that line)

**Pull request merged**